examples/demo/vanderwaals.py

```diff
@@ -0,0 +1,101 @@
+"""
+Example application for exploring Van der Waal's Equation
+"""
+
+from traits.api import (
+    Array,
+    Enum,
+    Float,
+    HasTraits,
+    Instance,
+    observe,
+    Property,
+    Range,
+)
+from traitsui.api import Item, UItem, View
+from chaco.api import ArrayPlotData, Plot
+from enable.api import ComponentEditor
```

Comment:
nit-pick : for users of ETS, i think all of ETS comes under the same category so `traits` and `traitsui` come after `chaco` and `enable`. Also, numpy needs to come before `traits`.

nit-nit-pick : i don't understand why we do `from numpy import arange` where as the community normally does `import numpy as np; np.arange`.